backend/ee/danswer/external_permissions/google_drive/doc_sync.py

```diff
@@ -72,25 +71,6 @@ def _fetch_permissions_paginated(
 ) -> Iterator[dict[str, Any]]:
     next_token = None
 
```

Comment:
Removing this check because if the connector is indexing trashed documents, then permissions should be applied to those documents. 

If we want to filter out trashed documents, it should be done at the connector level